### PR TITLE
Fixes #29734 - Fix bar chart on config report page

### PR DESCRIPTION
**Pull request by xprazak2**

### CODE REVIEW

**Comment by tbrisker** [comment]
Much better, but looks like the chart is still overflowing a bit:
image
Also pushed a rebase on top of latest develop for you

**Comment by ekohl** [comment]
I first thought it was a duplicate of https://projects.theforeman.org/issues/26090, but looks like it's just a similar issue on a different page.

**Comment by xprazak2** [comment]
Thanks, donut chart should be now contained within the card as well.

**Review by tbrisker — APPROVED** [review]
Thanks @xprazak2 !

File: webpack/assets/javascripts/react_app/components/ConfigReports/ConfigReports.js [hunk 1]
```diff
@@ -38,15 +40,15 @@ const ConfigReports = props => {
...
+      <Col md={5} className="bar-chart-medium-width">
         <ChartBox
           {...chartBoxProps}
           type="bar"
           chart={{ data: statusChartData, id: 'report-status' }}
           title={__('Report Status')}
         />
       </Col>
-      <Col md={4}>
+      <Col md={2}>
```
Comment (ekohl) [inline webpack/assets/javascripts/react_app/components/ConfigReports/ConfigReports.js:51]: This makes it very hard to read on a smaller screen:
Screen Shot 2020-06-01 at 15 53 36

**Pull request merged**